### PR TITLE
Consistency for APIs in Kyber Storage

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

File: contracts/sol6/KyberStorage.sol [hunk 1]
```diff
@@ -55,6 +55,7 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
     );
 
     event ListReservePairs(
+        bytes32 reserveId,
```
Comment (ilanDoron) [inline contracts/sol6/KyberStorage.sol:58]: can index here.

Comment (manhlx3006) [reply contracts/sol6/KyberStorage.sol]: Updated

File: contracts/sol6/KyberStorage.sol [hunk 2]
```diff
@@ -159,13 +160,16 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
 
     /// @notice Can be called only by operator
     /// @dev Removes a reserve from the network
-    /// @param reserve The reserve address
+    /// @param reserveId The reserve address
```
Comment (Anyhowclick) [inline contracts/sol6/KyberStorage.sol:163]: The reserve ID

File: contracts/sol6/KyberStorage.sol [hunk 3]
```diff
@@ -190,31 +194,32 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
...
 
         emit RemoveReserveFromStorage(reserve, reserveId);
-        return reserveId;
+        return true;
     }
 
     /// @notice Can be called only by operator
     /// @dev Allow or prevent a specific reserve to trade a pair of tokens
-    /// @param reserve The reserve address
+    /// @param reserveId The reserve address
```
Comment (Anyhowclick) [inline contracts/sol6/KyberStorage.sol:202]: Here as well

Comment (manhlx3006) [reply contracts/sol6/KyberStorage.sol]: updated

**Pull request merged**